// [python quickstart](https://grpc.io/docs/quickstart/python.html#run-a-grpc-application)
// python -m grpc_tools.protoc --python_out=. --grpc_python_out=. -I. helloworld.proto


syntax = "proto3";

service Greeter {
	rpc SayHello(HelloRequest) returns(HelloReplay) {}
	rpc SayHelloAgain(HelloRequest) returns(HelloReplay) {}

}


message HelloRequest {
	string name = 1;
}
message HelloReplay {
	string message = 1;
}